### PR TITLE
changed name to correct one

### PR DESCRIPTION
**Pull request by hector-norza**
I deleted older version of duplicate package

### CODE REVIEW

File: _data/releases/latest/js-packages.csv [hunk 1]
```diff
@@ -574,7 +574,6 @@
...
 "azure-arm-billing","2.2.0","","Resource Management - Billing","Billing","NA","NA","NA","mgmt","false","","","","deprecated","","","@azure/arm-billing","","","","Deprecated. Added for MSDocs ToC generation. Not published by azure-sdk automation."
 "@azure/arm-changeanalysis","1.1.1","","Resource Management - Change Analysis","Changeanalysis","changeanalysis","","NA","mgmt","true","","","","","3/31/2023","true","","https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md","","",""
-"@azure/arm-search","2.0.0","","Resource Management - Cognitive Search","Search","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/search/arm-search","","NA","mgmt","true","","","","deprecated","3/31/2023","true","@azure/arm-search","https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md","","",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:577]: We had this entry originally to track the last version of track 1 this should have New=false in the fields to indicate that. So before removing this we should check with @ronniegeraghty to see if we are still interested in having this for the track 1 version number in the inventory table or not.

Comment (ronniegeraghty) [reply _data/releases/latest/js-packages.csv:577]: We still want to track but the package name here is the package name for the Track 2 library. Matching the row we made a change to in the Display Name column. The package name for the track 1 version of this library is `azure-arm-search` which shows up in a different row.

Comment (weshaggard) [reply _data/releases/latest/js-packages.csv:577]: For JS and Python we didn't change the names of the package from track 1 and track 2 in a lot of cases so we had 2 entries for those to track the last version of that package name was track 1, which is what this entry is about. azure-arm-search is actually pre-track 1.

Comment (ronniegeraghty) [reply _data/releases/latest/js-packages.csv:577]: Okay Understood. It should end up being okay in the inventory dashboard as long as the `New` column is set to `false`.

Comment (ronniegeraghty) [reply _data/releases/latest/js-packages.csv:577]: Added line back and changed `New` column to `false`

**Review by ronniegeraghty — APPROVED** [review]

**Pull request merged**